minijanus.js

```diff
@@ -11,8 +11,8 @@ function JanusPluginHandle(session) {
 }
 
 /** Attaches this handle to the Janus server and sets its ID. **/
-JanusPluginHandle.prototype.attach = function(plugin) {
-  var payload = { plugin: plugin, "force-bundle": true, "force-rtcp-mux": true };
+JanusPluginHandle.prototype.attach = function(plugin, loop_index) {
+  var payload = { plugin: plugin, loop_index: loop_index, "force-bundle": true, "force-rtcp-mux": true };
```

Comment:
Is this working correctly on janus side if we give `loop_index: undefined`?
It's probably better to not include the key at all in the payload if it's undefined, less bytes to send :)
Do this:
```
if (loop_index) {
  payload.loop_index = loop_index;
}
```